app/serializers/serial_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class SerialSerializer < ActiveModel::Serializer
+  attributes :id, :title, :description, :thumbnail_url
+
+  has_many :posts
+
+  def thumbnail_url
+    scope.image_url(object.thumbnail_url)
```

Comment:
Yes. We need this logic for local development.
If we don't have asset_host, `thumbnail_url` will return only path :dizzy_face: